content/en/seasaltApi/seasalt-api/01-seachat-api-intro.md

```diff
@@ -11,6 +11,40 @@ menu:
 weight: 103
 toc: true
 ---
+<div style="display: flex; align-self: flex-end; align-items: baseline">
```

Comment:
this reminds me: can you add a section under "Resources" called "SeaChat APIs"? Then point to this page
![image](https://github.com/user-attachments/assets/554d11b5-e9d8-4fc0-889f-e09eeb73ef06)